app/src/main/res/layout/screen_registration_location_permission.xml

```diff
@@ -39,10 +39,9 @@
   <ImageView
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_above="@+id/registrationlocation_rationale_card"
-    android:layout_below="@+id/toolbar"
-    android:layout_marginBottom="-8dp"
-    android:scaleType="centerCrop"
+    android:layout_centerInParent="true"
+    android:adjustViewBounds="true"
+    android:scaleType="fitXY"
```

Comment:
`fitXY` can cause aspect ratio issues, correct?

Reply:
Oh I forgot to remove it, `android:adjustViewBounds` should take care of that so we don't actually need a `scaleType`.  I will remove it. But yeah `fitXY`, when used alone, can cause aspect ratio issues.